src/cgame/rocket/rocketKeyBinder.h

```diff
@@ -55,7 +55,7 @@ class RocketKeyBinder : public Rml::Core::Element, public Rml::Core::EventListen
 	{
 	}
 
-	void OnAttributeChange( const Rml::Core::ElementAttributes &changed_attributes )
+	void OnAttributeChange( const Rml::Core::ElementAttributes &changed_attributes ) override
```

Comment:
Most of those changes could have been done in the rocket code (would have fixed a bunch of warnings, too, and possibly could enable treating the lack of override as error).

Reply:
Note that I was referring to the *override* changes, in the commit they were introduced.